<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormHeaderRight</class>
 <widget class="QWidget" name="FormHeaderRight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>96</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>96</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>96</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>14</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>540</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>540</width>
       <height>75</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:none;
	max-width:110px;
	min-width:110px;
	max-height:74px;
	min-height:74px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonTools">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>74</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>74</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButtonTools{
	background:url(:/public/header/heder_option.png);
	border:none;
}
#pushButtonTools:pressed{
	background:url(:/public/header/header_option_clicked.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSetting">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>74</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>74</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image:url(:/public/header/header_setting.png)
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetAlarm" native="true">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>74</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>74</height>
         </size>
        </property>
        <widget class="QPushButton" name="pushButtonAlarm">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>110</width>
           <height>74</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>74</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>74</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelAlarm">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>0</y>
           <width>50</width>
           <height>74</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>74</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>74</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:36px;
color:#22d05a;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSilence">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>74</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>74</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButtonSilence{
	background-image:url(:/public/header/header_alarm.png);
}
#pushButtonSilence:pressed{
	background-image:url(:/public/header/header_alarm_clicked.png)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
